tests/test_hipcc.py

```diff
@@ -314,3 +324,13 @@ def test_host_device_compiler_backend(self):
                 f"DEVICE_COMPILER: {comb[1][0]} {comb[1][1]}, "
                 f"BACKENDS: {comb[2]}",
             )
+
+
+global param_map
+# set param_map, that filters expect the following parameters in the
+# order
```

Comment:
Could you rephrase this? I don't really understand what you are trying to say here.

Reply:
This was a left-over from debugging. I removed the code block.